docs/zh_cn/明细数据查询的典型Carbon应用-点查 过滤条件.md

```diff
@@ -0,0 +1,520 @@
+| 153      | 4.468                    | 7.873                    | 27.779                    |
+| 154      | 9.136                    | 6.559                    | 82.546                    |
+| 155      | 5.69                     | 16.854                   | 33.156                    |
+| 156      | 4.253                    | 6.499                    | 18.284                    |
+| 157      | 8.223                    | 5.983                    | 26.154                    |
+| 158      | 4.733                    | 16.357                   | 17.805                    |
+| 159      | 4.347                    | 15.032                   | 13.611                    |
+| 160      | 4.276                    | 13.735                   | 26.219                    |
+| 161      | 17.02                    | 13.094                   | 76.501                    |
+| 162      | 4.765                    | 22.683                   | 40.978                    |
+| 163      | 5.941                    | 12.033                   | 21.6                      |
+| 164      | 66.877                   | 11.795                   | 60.514                    |
+| 165      | 15.101                   | 6.563                    | 39.268                    |
+| 166      | 8.748                    | 9.21                     | 18.536                    |
+| 167      | 5.412                    | 10.025                   | 19.954                    |
+| 168      | 11.692                   | 8.565                    | 35.502                    |
+| 总计     | 1109.742                 | 1876.579                 | 5191.277                  |
+
+查询性能的对比
+
+这里使用了一组点查的SQL对使用本地字典，全局字典和不使用字典的德数据表进行了查询统计，记录一些统计结论。
+
+查询SQL分别使用
+
+```
+select * from test.detail_benchmark where MSISDN="13900000000" limit 2000;
+select * from test.detail_loacl_dict where MSISDN="13900000000" limit 2000;
+select * from test.detail_global_dict where MSISDN="13900000000" limit 2000;
+```
+
+记录的查询耗时情况如下：
+
+|                | detail_benchmark | detail_loacl_dict | detail_global_dict |
+| -------------- | ---------------- | ----------------- | ------------------ |
+| 查询五次平均值 | 18.5             | 14.543            | 7.750              |
+
+从结果可以看出，使用全局字典，点查字典字段用时最少，本地字典次之，无字典最差。
+
+用户在选择字典的使用上时，可以根据自身对数据加载的要求和查询要求来选择字典，本案例由于对数据入库的时效性有较高要求，为了避免数据加载的延迟，因此在建表的时候选择无字典的方案。
+
+
+
+### 建表 Create table 时SORT_COLUMNS和SORT_SCOPE的选择
+
+​        关于SORT_COLUMNS的配置可以参考CarbonData[官网](http://carbondata.apache.org/ddl-of-carbondata.html#create-table)的说明, 当用户不指定SORT_COLUMNS时，默认将不建立SORT_COLUMNS，当前的SORT_COLUMNS之支持：tring, date, timestamp, short, int, long, byte, boolean类型。SORT_COLUMNS 在用户指定的列上建立MKD索引，将有助于查询性能的提升，但是将稍微影响加载性能，因此只需要给需要的列上设置SORT_COLUMNS即可，无需给所有列都设置SORT_COLUMNS。
+
+​         [SORT_SCOPE](http://carbondata.apache.org/ddl-of-carbondata.html#create-table)参数用户在指定了SORT_COLUMNS后，数据家在时排序的范围，当前支持的范围有：
+
+NO_SORT:  对加载的数据不排序
+
+LOCAL_SORT: 加载的数据在本地排序 
+
+GLOBAL_SORT:  加载的数据全局排序，它提升了高并发下点查的效率，对加载的性能的影响较大。
+
+BATCH_SORT:  用的比较少
```

Comment:
please remove it, BATCH_SORT is deprecated

Reply:
done